packages/teams-js/src/private/chat.ts

```diff
@@ -21,61 +21,53 @@ export namespace chat {
       ensureInitialized(FrameContexts.content);
-      const sendPromise = sendAndHandleError('conversations.openConversation', {
-        title: openConversationRequest.title,
-        subEntityId: openConversationRequest.subEntityId,
-        conversationId: openConversationRequest.conversationId,
-        channelId: openConversationRequest.channelId,
-        entityId: openConversationRequest.entityId,
+      const sendPromise = sendAndHandleError('chat.openChat', {
+        members: openChatRequest.user,
+        message: openChatRequest.message,
       });
-      if (openConversationRequest.onStartConversation) {
-        registerHandler(
-          'startConversation',
-          (subEntityId: string, conversationId: string, channelId: string, entityId: string) =>
-            openConversationRequest.onStartConversation({
-              subEntityId,
-              conversationId,
-              channelId,
-              entityId,
-            }),
-        );
-      }
-      if (openConversationRequest.onCloseConversation) {
-        registerHandler(
-          'closeConversation',
-          (subEntityId: string, conversationId?: string, channelId?: string, entityId?: string) =>
-            openConversationRequest.onCloseConversation({
-              subEntityId,
-              conversationId,
-              channelId,
-              entityId,
-            }),
-        );
-      }
       resolve(sendPromise);
     });
   }
-
   /**
    * @hidden
    * Hide from docs
    * --------------
-   * Allows the user to close the conversation in the right pane
+   * Allows the user to create a chat with multiple users (2+) and allows
+   * for the user to specify a message and name the topic of the conversation. If
+   * only 1 user is provided into users array default back to origin openChat.
+   *
+   * @returns Promise resolved upon completion
    */
-  export function closeConversation(): void {
-    ensureInitialized(FrameContexts.content);
-    sendMessageToParent('conversations.closeConversation');
-    removeHandler('startConversation');
-    removeHandler('closeConversation');
+  export function openGroupChat(openChatRequest: OpenGroupChatRequest): Promise<void> {
+    return new Promise<void>(resolve => {
+      if (openChatRequest.users.length < 1) {
+        throw Error('OpenGroupChat Failed: No users specified');
+      }
+      if (openChatRequest.users.length === 1) {
+        const chatRequest: OpenSingleChatRequest = {
+          user: openChatRequest.users[0],
+          message: openChatRequest.message,
+        };
+        openChat(chatRequest);
+      } else {
+        ensureInitialized(FrameContexts.content);
+        const sendPromise = sendAndHandleError('chat.openChat', {
+          members: openChatRequest.users,
+          message: openChatRequest.message,
+          topic: openChatRequest.topic,
+        });
+        resolve(sendPromise);
+      }
+    });
   }
-
   /**
    * @hidden
    * Hide from docs
```

Comment:
Looks like you didn't change this, but the callback param for getChatMembers doesn't exist anymore we should probably delete this part of the comment